.circleci/config.yml

```diff
@@ -11,7 +11,7 @@ jobs:
           steps:
             - run:
                 name: 'Install'
-                command: npm install
+                command: npm install --frozen-lockfile --non-interactive
```

Comment:
shouldn't this be yarn?

Reply:
I had a an error "segmentation whatever problem"...
Switched it back to npm after not being able to fix it. :)

Postponed it for later.